apps/angular-thirty-seconds/src/app/create-snippet/create-snippet.component.ts

```diff
@@ -29,8 +29,8 @@ function validatorMaxTags(maximumTags: number) {
 })
 export class CreateSnippetComponent {
 
-  @ViewChild('tagInput') tagInput: ElementRef<HTMLInputElement>;
-  @ViewChild('auto') matAutocomplete: MatAutocomplete;
+  @ViewChild('tagInput', { static: true }) tagInput: ElementRef<HTMLInputElement>;
```

Comment:
Why `true`?

Reply:
@thekiba Breaking changes in Angular v8 in preparation for v9: https://angular.io/guide/static-query-migration

Reply:
@thekiba After re-reading the docs I realize a `{ static: false }` is the recommended flag. I'll update it.